.vib/kibana/cypress/cypress/integration/kibana_spec.js

```diff
@@ -0,0 +1,88 @@
+/// <reference types="cypress" />
+
+import {
+  random,
+  closeThePopups,
+  skipTheWelcomeScreen,
+} from './utils'
+
+before(() => {
+  skipTheWelcomeScreen();
+})
+
+it('allows uploading a file',() => {
+  cy.get('a[data-test-subj="uploadFile"]').should('be.visible').click();
+  cy.get('input#filePicker').selectFile('cypress/fixtures/test-data-to-import.ndjson', {force:true});
+  cy.get('[data-test-subj="dataVisualizerFileOpenImportPageButton"]',).should('be.visible').click();
+  cy.fixture('testdata').then((td) => {
+    cy.get('[data-test-subj="dataVisualizerFileIndexNameInput"]').type(`${td.testIndexName}.${random}`);
+  });
+  cy.get('[data-test-subj="dataVisualizerFileImportButton"]').should('be.visible').click();
+  cy.get('[title="Step 2: Index created is complete"]').scrollIntoView().should('be.visible');
+  cy.get('[title="Step 3: Data uploaded is complete"').scrollIntoView().should('be.visible');
+  cy.get('[title="Step 4: Index pattern created is complete"]').should('be.visible');
```

Comment:
If possible, you can use `[title*="Index created is complete"]` instead. This is somewhat more generic and will work even if the step number is modified in upcoming versions.